client/src/components/Hamburger/Hamburger.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBars, faTimes } from '@fortawesome/free-solid-svg-icons';
```

Comment:
Rather than using these icons, can you use the `Icon` component

Reply:
Once you use the `Icon` component be sure to remove this code from the component.

Reply:
@ZakkMan , I'm not there yet since I'm still trying to work on making everything function properly

Reply:
The Icon component is there! Chat with Kelly if you have questions on how it works